keyboards/bastardkb/scylla/keymaps/cykedev/rules.mk

```
@@ -0,0 +1,17 @@
+MCU                 = atmega32u4
+BOOTLOADER          = atmel-dfu
+
+BOOTMAGIC_ENABLE    = yes     # Enable Bootmagic Lite
+MOUSEKEY_ENABLE     = no      # Mouse keys
+EXTRAKEY_ENABLE     = yes     # Audio control and System control
+CONSOLE_ENABLE      = no      # Console for debug
+COMMAND_ENABLE      = no      # Commands for debug and configuration
+SLEEP_LED_ENABLE    = no      # Breathing sleep LED during USB suspend
+NKRO_ENABLE         = yes     # USB Nkey Rollover
+BACKLIGHT_ENABLE    = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE     = no      # Enable keyboard RGB underglow
+RGB_MATRIX_ENABLE   = no      # Enable keyboard RGB matrix (do not use together with RGBLIGHT_ENABLE)
+BLUETOOTH_ENABLE    = no      # Enable Bluetooth
+AUDIO_ENABLE        = no      # Audio output
+KEY_OVERRIDE_ENABLE = yes     # Enable Key Overrides, see https://docs.qmk.fm/#/feature_key_overrides
+SPLIT_KEYBOARD      = yes     # Split Keyboard
```

You only need to include what the keyboard doesn't have set the same. 

```suggestion
MOUSEKEY_ENABLE     = no      # Mouse keys
NKRO_ENABLE         = yes     # USB Nkey Rollover
RGB_MATRIX_ENABLE   = no      # Enable keyboard RGB matrix
KEY_OVERRIDE_ENABLE = yes     # Enable Key Overrides, see https://docs.qmk.fm/#/feature_key_overrides
```